app/assets/stylesheets/admin/base.scss

```diff
@@ -8,6 +8,22 @@
 @import "_shims";
 @import "_typography";
 
+@import "design-patterns/breadcrumbs";
+
+.govuk-breadcrumbs {
```

Comment:
I'm not sure if it's still used, but I think we should remove this now.

We added separate css here: https://github.com/alphagov/whitehall/blob/541cea6d403c069916a4d0a28acca74f3a2e07a9/app/assets/stylesheets/admin/topics_section.scss
which we should be able to reuse for the breadcrumb preview.